pom.xml

```diff
@@ -670,13 +707,30 @@
 		<dependency>
 			<groupId>io.jenkins.plugins</groupId>
 			<artifactId>javax-activation-api</artifactId>
-			<version>1.2.0-3</version> <!-- TODO use version from BOM when possible -->
 		</dependency>
 
 		<dependency>
 			<groupId>io.jenkins.plugins</groupId>
 			<artifactId>jaxb</artifactId>
-			<version>2.3.6-1</version> <!-- TODO use version from BOM when possible -->
+		</dependency>
+		<!-- Explicit JAXB API -->
+		<dependency>
+			<groupId>jakarta.xml.bind</groupId>
+			<artifactId>jakarta.xml.bind-api</artifactId>
+			<version>2.3.2</version>
+		</dependency>
+
+		<!-- Explicit JAXB Runtime -->
+		<dependency>
+			<groupId>org.glassfish.jaxb</groupId>
+			<artifactId>jaxb-runtime</artifactId>
+			<version>2.3.2</version>
+		</dependency>
+
+		<dependency>
+			<groupId>commons-fileupload</groupId>
+			<artifactId>commons-fileupload</artifactId>
+			<version>1.5</version>
 		</dependency>
```

Comment:
This won't work soon on Java EE 9 (coming to a Jenkins weekly later this year). Better to upgrade to FileUpload 2.x, which has both EE 8 and EE 9 support.

Reply:
Sure. The general recommendation is not to depend on FileUpload directly but to invoke the Jenkins API `StaplerRequest#getFileItem[2]` which will invoke the correct FileUpload methods for `javax` or `jakarta` servlet imports. If you need to do file processing yourself, it may be desirable to rewrite it to use the multipart file upload API that is built into newer versions of Java EE rather than using a library like Commons FileUpload.